mats.js

```diff
@@ -42,11 +42,10 @@ export class Mat {
     RNOpencv3.setTo(this, color)
   }
 
-  get = async(rownum, colnum, data) => {
-    // not sure if data needs to be returned here ...
-    // this has not been tested yet ...
-    data = await RNOpencv3.getMatData(this, rownum, colnum)
-  }
+  get = async(colnum, rownum) => {
```

Comment:
Creo que sería más legible si los parámetros de esta función se llamaran x, y, encapsulando lo de rownum, colnum por debajo
```suggestion
  get = async(x, y) => {
```